src/makeInfinityAdder.js

```diff
@@ -6,7 +6,28 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  const numbers = [];
```

Comment:
no need to save all numbers, just count how many times the function adder was called

Reply:
sorry, not count(another task), sum, you need to store the sum